torch/lib/THD/base/Cuda.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#ifdef WITH_CUDA
+#include <THC/THC.h>
+
+extern THCState** _THDCudaState;
+
+inline THCState* THDGetCudaState() {
+  return *_THDCudaState;
+}
+
+int THDGetStreamId(cudaStream_t stream);
+
+#include "Cuda.h"
```

Comment:
It's a header that's included in other files. Functions form `.h` should also be exposed here